pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -339,6 +339,12 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(ctx context.Context,
 			return reconcile.Result{RequeueAfter: timeout}, nil
 		}
 
+		if volume.VolumeType != string(cnstypes.CnsVolumeTypeFile) {
+			msg := fmt.Sprintf("CNS Volume: %s is not FILE volume", volumeID)
```

Comment:
i think we're using `logger.LogNewErrorf()`

Reply:
Also `FILE` -> `RWX`

Reply:
Done